br.com.eduk.yml

```diff
@@ -0,0 +1,60 @@
+app-id: br.com.eduk
```

Comment:
How are you related with https://app.eduk.com.br/ ? I don't see any mention of this software on that webpage.

Reply:
I serve as the product leader in this company and have recently uploaded a new update of the project to the EduK company's repository. This action strengthens my connection with the organization and demonstrates ongoing commitment to the development and enhancement of our solutions.